### ISSUE TITLE
The link in debug html mode does not work correctly the basehref is not good.

### ISSUE THREAD

**Issue by lionel-meunier** [dfederm/karma-jasmine-html-reporter#44]
In debug mode (on the debug.html page in chrome)

The link is created with this type of url :

``
<a title="should be created" href="?spec=ConfirmService%20should%20be%20created">should be created</a>
``
While the base href is at "/"

Replace the construction of the link with

``
<a title="should be created" href="debug.html?spec=ConfirmService%20should%20be%20created">should be created</a>
``

Replace method toQueryString by 

````
function toQueryString(paramMap) {
    var qStrPairs = [];
    for (var prop in paramMap) {
      qStrPairs.push(
        encodeURIComponent(prop) + '=' + encodeURIComponent(paramMap[prop])
      );
    }
    var preffix = '';
    if(window.location.pathname === '/debug.html') {
      preffix = 'debug.html';
    }
    return preffix + '?' + qStrPairs.join('&');
  }
````

**Comment by dfederm** [comment]
Please read the readme: https://github.com/dfederm/karma-jasmine-html-reporter#develop

This file comes from the jasmine package.

**Comment by jlpstolwijk** [comment]
should now be fixed in jasmine, see [here](https://github.com/jasmine/jasmine/issues/1906)

**Comment by dfederm** [comment]
Pulled in latest from jasmine and published v1.7.0. Seems to have fixed this issue.
